### PR TITLE
upgrading to 34

### PR DESCRIPTION
**Pull request by prateekbh**
- upgrades to 0.34

### CODE REVIEW

**Comment by artipost** [comment]
Failed pictures (24): [failed-pictures.zip](https://gist.githubusercontent.com/artipost/c0b32ea39107d4db9e8b8b44f6cadcca/raw/4a3f6025e5ccfe12003ba53004cf7d55d6adbb3f/failed-pictures.zip)

##
:postbox: Posted by [artipost.io](https://artipost.io), through Travis build [364378455](https://travis-ci.org/prateekbh/preact-material-components/builds/364378455), for commit [a3868ab](https://github.com/prateekbh/preact-material-components/pull/744/commits/a3868ab3354cc3769c692b04cc3684d207c68b18).

File: Select/Select.jsx [hunk 1]
```diff
@@ -49,48 +49,29 @@ class Select extends MaterialComponent {
...
-    props = Object.assign(baseProps, props);
-    if ('disabled' in props) {
-      delete props['disabled'];
-    }
-    if ('selected' in props) {
-      delete props['selected'];
-    }
-
-    return super.materialDom(props);
+    return <option value="grains">{props.children}</option>;
```
Comment (YePpHa) [inline Select/Select.jsx]: is the value supposed to be "grains"?

File: TextField/TextField.jsx [hunk 2]
```diff
@@ -74,6 +74,9 @@ class TextFieldInput extends MaterialComponent {
   componentWillUnmount() {
     this.MDComponent && this.MDComponent.destroy && this.MDComponent.destroy();
   }
+  value() {
```
Comment (YePpHa) [inline TextField/TextField.jsx]: Wouldn't it be better to name it either `getValue()` or make it a property with a getter method?

Comment (prateekbh) [reply TextField/TextField.jsx]: a value prop is already supported to be mapped to native element but yes making it `getValue`

File: Select/Select.jsx (hunk 1 above)
Comment (prateekbh) [reply Select/Select.jsx]: fixed

**Comment by prateekbh** [comment]
@YePpHa fixed the comments

**Comment by artipost** [comment]
Failed pictures (1): [failed-pictures.zip](https://gist.githubusercontent.com/artipost/8f499a095c1037ddbee3d8c2f5b979df/raw/d59f8186ad908929e6ae8e8715d866ba3a21b204/failed-pictures.zip)

##
:postbox: Posted by [artipost.io](https://artipost.io), through Travis build [365839775](https://travis-ci.org/prateekbh/preact-material-components/builds/365839775), for commit [7d92211](https://github.com/prateekbh/preact-material-components/pull/744/commits/7d92211a19c5c2b1bd7e5719d6644ca402e14332).

**Pull request merged**